### DISCUSSION TITLE
Maven URL listed on docs not working

### DISCUSSION THREAD

**Discussion by Aahan-Patel** [MoreMcmeta/core#69] [Help Wanted] [answered]
Hello, I was trying to add the Maven URL to my build.gradle file however when building it, gradle cannot resolve it.  Additionally inputting the URL into my search engine returns a 404 error.

**Comment by soir20** [comment, marked as answer]
Are you using the GitHub Packages Maven? Or are you using Curse or Modrinth Maven?

If you're using GitHub Packages, do you have the Maven username and password specified? See the docs [here](https://github.com/MoreMcmeta/core/wiki/Plugin-Docs:-Maven#example-setup).

**Reply by Aahan-Patel** [reply]
I got it to work. Seemed to be a problem on my end. Thanks for the help
